composition-js/src/composeDirectiveManager.ts

```diff
@@ -174,6 +175,7 @@ export class ComposeDirectiveManager {
       this.pushHint(new CompositionHint(
         HINTS.DIRECTIVE_COMPOSITION_INFO,
         `Directive "@${directive.name}" should not be explicitly manually composed since it is a federation directive composed by default`,
+        directive.name,
```

Comment:
nit: Directive schema coordinates should start with `@`

Reply:
Good catch. Updated in latency revision